OpenRA.Game/Map/MapGeneratorSettings.cs

```diff
@@ -0,0 +1,364 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace OpenRA
+{
+	public sealed class MapGeneratorSettings
```

Comment:
Neither this or `IMapGenerator` seem to be used in OpenRA.Game, so can we please move both into Mods.Common?